falcon/asgi/ws.py

```diff
@@ -521,12 +539,13 @@ class WebSocketOptions:
             (default ``1011``). For a list of valid close codes and ranges,
             see also: https://tools.ietf.org/html/rfc6455#section-7.4
         default_close_reasons (dict): A default mapping between the Websocket
-            close code and the reason why the connection is close. Close codes
-            corresponding to HTTPErrors are not included as they will be rendered
-            automatically using HTTP status.
-        media_handlers (dict): A dict-like object for configuring media handlers
-            according to the WebSocket payload type (TEXT vs. BINARY) of a
-            given message. See also: :ref:`ws_media_handlers`.
+            close code and the reason why the connection is close.
+            Close codes corresponding to HTTP errors are also included in this
+            mapping.
+        media_handlers (dict): A dict-like object for configuring media
+            handlers according to the WebSocket payload type
+            (``TEXT`` vs. ``BINARY``) of a given message.
+            See also: :ref:`ws_media_handlers`.
```

Comment:
I don't see a reason to change these descriptions here. I've modified them here https://github.com/falconry/falcon/pull/2290

Reply:
Aha, missed that. I'll roll back this part to minimize merge conflicts.

Reply:
Rolled back. But I still need to incrementally add `default_close_reasons` here.